src/Html.php

```diff
@@ -1878,7 +1826,10 @@ public static function footer($keepDB = false)
 
         TemplateRenderer::getInstance()->display('layout/parts/page_footer.html.twig', $tpl_vars);
 
-        self::displayDebugInfos();
+        if (!str_starts_with($_SERVER['PHP_SELF'], $CFG_GLPI['root_doc'] . '/install/')) {
```

Comment:
It does not detect the update process that is shown in replacement of any page. In such case `$_SERVER['PHP_SELF']` correspond to the expected page (e.g. `/index.php`, `/front/central.php`, ...).

![image](https://github.com/glpi-project/glpi/assets/33253653/53965363-c79f-45ac-bbba-ba33e50aaea0)

Reply:
How much of an issue would that be? With the actual install/upgrade page, the user was forced into debug mode even if they had no permission to have access to it. Here though, it seems like it only shows for users with access that enabled it.